docs/use/formatting_outputs.md

```diff
@@ -172,6 +217,49 @@ and even internal images can be rendered!
 display(Markdown('![figure](../_static/logo-wide.svg)'))
 ```
 
+But setting the `render_markdown_format` to `myst` will allow for more advanced formatting,
```

Comment:
given that this package is explicitly labeled as "myst", why not enable myst markdown parsing by default?